Please take a look at the torch test failure:

> FAILED keras/src/layers/core/einsum_dense_test.py::EinsumDenseTest::test_quantize_with_specific_equations_int4_btd,df->btf - AssertionError: tensor(0.0029, grad_fn=<MeanBackward1>) not less than 0.002

--- a/keras/src/layers/core/einsum_dense_test.py
+++ b/keras/src/layers/core/einsum_dense_test.py
@@ -553,7 +553,7 @@ class EinsumDenseTest(testing.TestCase):
             "btd,df->btf",
             (None, 4),
             (1, 2, 4),
-            2e-3,
+            3e-3,
         ),
     )
     def test_quantize_with_specific_equations(

